policyengine_us/parameters/gov/states/nh/tax/credits/education_tax_credit/rate.yaml

```
@@ -0,0 +1,21 @@
+description: New Hampshire matches this share of Education Tax Credit, depending on the donation.
+metadata:
+  threshold_unit: currency-USD
+  rate_unit: /1
+  threshold_period: year
+  type: marginal_amount
+  label: New Hampshire Education Tax Credit 
+  reference: 
+    - title: TITLE V- TAXATION - CHAPTER 77-G - EDUCATION TAX CREDIT - Section 77-G:4
+    - href: https://www.gencourt.state.nh.us/rsa/html/V/77-G/77-G-4.htm 
```

Also this source doesn't support the numbers here
```suggestion
      href: https://www.gencourt.state.nh.us/rsa/html/V/77-G/77-G-4.htm 
```